C-Plus-Plus/cp/Reverse_string_using_stack.cpp

```diff
@@ -0,0 +1,58 @@
+/* 
+Given a string,print reverse of the string using stack
+Approach:-
+In stack,we know LIFO principle works.
+First,i create an empty stack,then i push each character of string in stack
+then,using for loop i popped out every single character.
+*/
```

Comment:
Use multiline comments Don't use single line comments, when you wrote n line of comments.